examples/user_profiles/connector.py

```diff
@@ -0,0 +1,144 @@
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+from fivetran_connector_sdk import Logging as log
+import requests as rq
+import pandas as pd
+
+
+def schema (configuration: dict):
+    return [
+        {
+            "table": "profile",
+            "primary_key": ["id"],
+            "columns": {
+                "id": "INT",
+                "gender": "STRING",
+                "fullName": "STRING",
+                "email": "STRING",
+                "age": "INT",
+                "date": "STRING",
+                "mobile": "STRING",
+                "nationality": "STRING"
+            }
+
+        },
+        {
+            "table": "location",
+            "primary_key": ["profileId"],
+            "columns": {
+                "profileId": "INT",
+                "street": "STRING",
+                "city": "STRING",
+                "state": "STRING",
+                "country": "STRING",
+                "postcode": "STRING"
+            }
+        },
+        {
+            "table": "Login",
```

Comment:
this should be `login`

Reply:
This isn't necessarily invalid. How would we handle this in a Java connector? We would apply renaming rules right?

Reply:
Doesn't core apply the renaming rules?
I don't think we should cover that edge case in this examples - I would support renaming this in the example to 'login' to avoid the question here.